app/scripts/directives/overview/listRow.js

```diff
@@ -66,7 +66,6 @@
 
     var updateCurrent = function(apiObject) {
       if (!apiObject ||
-          row.current ||
```

Comment:
This check prevent `row.current` from being updated to the new value for `apiObject`, which mean the donut was showing the old desired replicas value forever.